libraries/nginx_service.rb

```diff
@@ -94,6 +96,7 @@ def action_enable
       directory '/run/nginx' do
```

Comment:
It may make more sense for us to use /var/run on older platforms. I think that /run is a systemd-ism so is likely only applicable on ubuntu-16.04 and rhel-7.1 (and above).